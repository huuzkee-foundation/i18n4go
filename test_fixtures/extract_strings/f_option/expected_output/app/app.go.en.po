# filename: f_option/input_files/app.go, offset: 1289, line: 34, column: 16
msgid "help"
msgstr "help"

# filename: f_option/input_files/app.go, offset: 1372, line: 37, column: 28
msgid "%s help [COMMAND]"
msgstr "%s help [COMMAND]"

# filename: f_option/input_files/app.go, offset: 1710, line: 50, column: 59
msgid "VERSION:"
msgstr "VERSION:"

# filename: f_option/input_files/app.go, offset: 2455, line: 79, column: 48
msgid "CF_NAME"
msgstr "CF_NAME"

# filename: f_option/input_files/app.go, offset: 35, line: 3, column: 23
msgid "{{.Title \"NAME:\"}}\n   {{.Name}} - {{.Usage}}\n\n{{.Title \"USAGE:\"}}\n   [environment variables] {{.Name}} [global options] command [arguments...] [command options]\n\n{{.Title \"VERSION:\"}}\n   {{.Version}}\n\n{{.Title \"BUILD TIME:\"}}\n   {{.Compiled}}\n   {{range .Commands}}\n{{.SubTitle .Name}}{{range .CommandSubGroups}}\n{{range .}}   {{.Name}} {{.Description}}\n{{end}}{{end}}{{end}}\n{{.Title \"ENVIRONMENT VARIABLES\"}}\n   CF_COLOR=false                     Do not colorize output\n   CF_HOME=path/to/dir/               Override path to default config directory\n   CF_STAGING_TIMEOUT=15              Max wait time for buildpack staging, in minutes\n   CF_STARTUP_TIMEOUT=5               Max wait time for app instance startup, in minutes\n   CF_TRACE=true                      Print API request diagnostics to stdout\n   CF_TRACE=path/to/trace.log         Append API request diagnostics to a log file\n   HTTP_PROXY=proxy.example.com:8080  Enable HTTP proxying for API requests\n\n{{.Title \"GLOBAL OPTIONS\"}}\n   --version, -v                      Print the version\n   --help, -h                         Show help\n"
msgstr "{{.Title \"NAME:\"}}\n   {{.Name}} - {{.Usage}}\n\n{{.Title \"USAGE:\"}}\n   [environment variables] {{.Name}} [global options] command [arguments...] [command options]\n\n{{.Title \"VERSION:\"}}\n   {{.Version}}\n\n{{.Title \"BUILD TIME:\"}}\n   {{.Compiled}}\n   {{range .Commands}}\n{{.SubTitle .Name}}{{range .CommandSubGroups}}\n{{range .}}   {{.Name}} {{.Description}}\n{{end}}{{end}}{{end}}\n{{.Title \"ENVIRONMENT VARIABLES\"}}\n   CF_COLOR=false                     Do not colorize output\n   CF_HOME=path/to/dir/               Override path to default config directory\n   CF_STAGING_TIMEOUT=15              Max wait time for buildpack staging, in minutes\n   CF_STARTUP_TIMEOUT=5               Max wait time for app instance startup, in minutes\n   CF_TRACE=true                      Print API request diagnostics to stdout\n   CF_TRACE=path/to/trace.log         Append API request diagnostics to a log file\n   HTTP_PROXY=proxy.example.com:8080  Enable HTTP proxying for API requests\n\n{{.Title \"GLOBAL OPTIONS\"}}\n   --version, -v                      Print the version\n   --help, -h                         Show help\n"

# filename: f_option/input_files/app.go, offset: 1312, line: 35, column: 16
msgid "h"
msgstr "h"

# filename: f_option/input_files/app.go, offset: 1332, line: 36, column: 16
msgid "Show help"
msgstr "Show help"

# filename: f_option/input_files/app.go, offset: 1673, line: 50, column: 22
msgid "\n%s\n%s\n\n"
msgstr "\n%s\n%s\n\n"

# filename: f_option/input_files/app.go, offset: 1873, line: 57, column: 36
msgid "Jan 2, 2006 3:04PM"
msgstr "Jan 2, 2006 3:04PM"

