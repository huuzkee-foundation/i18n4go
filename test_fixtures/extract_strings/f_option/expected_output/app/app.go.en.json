[
    {
        "id": "help",
        "translation": "help"
    },
    {
        "id": "%s help [COMMAND]",
        "translation": "%s help [COMMAND]"
    },
    {
        "id": "VERSION:",
        "translation": "VERSION:"
    },
    {
        "id": "CF_NAME",
        "translation": "CF_NAME"
    },
    {
        "id": "{{.Title \"NAME:\"}}\n   {{.Name}} - {{.Usage}}\n\n{{.Title \"USAGE:\"}}\n   [environment variables] {{.Name}} [global options] command [arguments...] [command options]\n\n{{.Title \"VERSION:\"}}\n   {{.Version}}\n\n{{.Title \"BUILD TIME:\"}}\n   {{.Compiled}}\n   {{range .Commands}}\n{{.SubTitle .Name}}{{range .CommandSubGroups}}\n{{range .}}   {{.Name}} {{.Description}}\n{{end}}{{end}}{{end}}\n{{.Title \"ENVIRONMENT VARIABLES\"}}\n   CF_COLOR=false                     Do not colorize output\n   CF_HOME=path/to/dir/               Override path to default config directory\n   CF_STAGING_TIMEOUT=15              Max wait time for buildpack staging, in minutes\n   CF_STARTUP_TIMEOUT=5               Max wait time for app instance startup, in minutes\n   CF_TRACE=true                      Print API request diagnostics to stdout\n   CF_TRACE=path/to/trace.log         Append API request diagnostics to a log file\n   HTTP_PROXY=proxy.example.com:8080  Enable HTTP proxying for API requests\n\n{{.Title \"GLOBAL OPTIONS\"}}\n   --version, -v                      Print the version\n   --help, -h                         Show help\n",
        "translation": "{{.Title \"NAME:\"}}\n   {{.Name}} - {{.Usage}}\n\n{{.Title \"USAGE:\"}}\n   [environment variables] {{.Name}} [global options] command [arguments...] [command options]\n\n{{.Title \"VERSION:\"}}\n   {{.Version}}\n\n{{.Title \"BUILD TIME:\"}}\n   {{.Compiled}}\n   {{range .Commands}}\n{{.SubTitle .Name}}{{range .CommandSubGroups}}\n{{range .}}   {{.Name}} {{.Description}}\n{{end}}{{end}}{{end}}\n{{.Title \"ENVIRONMENT VARIABLES\"}}\n   CF_COLOR=false                     Do not colorize output\n   CF_HOME=path/to/dir/               Override path to default config directory\n   CF_STAGING_TIMEOUT=15              Max wait time for buildpack staging, in minutes\n   CF_STARTUP_TIMEOUT=5               Max wait time for app instance startup, in minutes\n   CF_TRACE=true                      Print API request diagnostics to stdout\n   CF_TRACE=path/to/trace.log         Append API request diagnostics to a log file\n   HTTP_PROXY=proxy.example.com:8080  Enable HTTP proxying for API requests\n\n{{.Title \"GLOBAL OPTIONS\"}}\n   --version, -v                      Print the version\n   --help, -h                         Show help\n"
    },
    {
        "id": "h",
        "translation": "h"
    },
    {
        "id": "Show help",
        "translation": "Show help"
    },
    {
        "id": "\n%s\n%s\n\n",
        "translation": "\n%s\n%s\n\n"
    },
    {
        "id": "Jan 2, 2006 3:04PM",
        "translation": "Jan 2, 2006 3:04PM"
    }
]
